src/Doctrine/DBAL/Types/UuidArrayType.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Contao\CoreBundle\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Types\BlobType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+/**
+ * Type that maps an array of UUIDs to a BLOB SQL type.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ * @link https://blog.vandenbrand.org/2015/06/25/creating-a-custom-doctrine-dbal-type-the-right-way/
+ */
+class UuidArrayType extends BlobType
+{
+    const UUID_ARRAY = 'uuid_array';
+
+    /**
+     * Converts the serialized array of binary UUIDs to string representation.
+     *
+     * @param mixed            $value
+     * @param AbstractPlatform $platform
+     *
+     * @return string
+     */
+    public function convertToPhpValue($value, AbstractPlatform $platform)
+    {
+        if (null === $value || !is_string($value)) {
+            return null;
+        }
+
+        $value = @unserialize($value);
```

Comment:
> Theoretically, but `deserialize` is a method from the old framework which is not (necessarily) available here.

It is not available here ;)

> Also, if you would use Doctrine serialized array storage, the same problem would be the case.

That's a good point.

> Otherwise we would need to add a `secure_array` type that uses the same code as our deserialize method…?

No, it would be to specific for Contao then IMHO.